packages/upstash-redis/src/index.ts

```diff
@@ -0,0 +1,209 @@
+import type { Account as AdapterAccount } from "next-auth"
+import type {
+  Adapter,
+  AdapterUser,
+  AdapterSession,
+  VerificationToken,
+} from "next-auth/adapters"
+import type { Upstash } from "@upstash/redis/src/types"
+import * as uuid from "uuid"
+
+export interface UpstashRedisAdapterOptions {
+  baseKeyPrefix?: string
+  accountKeyPrefix?: string
+  accountByUserIdPrefix?: string
+  emailKeyPrefix?: string
+  sessionKeyPrefix?: string
+  sessionByUserIdKeyPrefix?: string
+  userKeyPrefix?: string
+  verificationTokenKeyPrefix?: string
+}
+
+const defaultOptions = {
```

Comment:
```suggestion
export const defaultOptions = {
```
Let's make it easy to reuse if necessary.